Relativity.Processing.SDK.md

```diff
@@ -4,20 +4,28 @@
 
 This package contains interfaces for the public APIs of the Relativity Processing application.
 
-## 11.3.101.10
+## 12.2.13
 
 ### Release Notes
 
-* Initial version
+* Bumped the Invariant Services package for ARM statistics endpoint defect fix. Fixed error field removals.
+* Added a NuGet package for Password Bank, renamed Password Bank tab, and moved from ApplicationBase.
+* Resolved dependency injection issues and removed low-use RSAPI code, along with legacy test projects.
+* Removed redundant Kepler endpoint check for ValidateApiEndpoints API, removed ProcessingBase package and consolidated projects into Relativity.Eca.
+* Added a new private endpoint for updating count after Post Publish Delete job.
+* Use Object Manager instead of RSAPI for Custodian, Processing Set and Processing Data Source updates.
+* Started using new Shared.ApplicationBase package with contract changes to IContextContainer.
+* Removed the Processing Profile Manager API endpoint, consumers should now use Object Manager API.

```

Comment:
Could also mention that all calls to Invariant WCF endpoints have been replaced with Kepler equivalents. No WCF is used by the Processing APIs whatsoever now.